pamelalozano/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,10 @@
 
 package com.google.sps.servlets;
 
+import java.util.*;  
```

Comment:
Prefer listing out specific imports to using the wildcard. (Wildcard imports are forbidden at Google)

Reply:
Perfect! Thanks